spec/request/connected_applications_request_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   context 'with valid response from okta' do
-    it 'should return list of grants by app' do
+    xit 'should return list of grants by app' do
```

Comment:
also here

Reply:
Looks like @kreek may have fixed it in his current PR, if you can merge in his fix instead of disabling.

Reply:
Disabling is fine for now to get production fixed.